includes/Modules/Sign_In_With_Google.php

```diff
@@ -45,6 +46,48 @@ final class Sign_In_With_Google extends Module implements Module_With_Assets, Mo
 	 */
 	const MODULE_SLUG = 'sign-in-with-google';
 
+	/**
+	 * Google_Client instance.
+	 *
+	 * @since n.e.x.t
+	 * @var Google_Client
+	 */
+	protected $client;
+
+	/**
+	 * Option name for persistent user ID.
+	 *
+	 * @since n.e.x.t
+	 * @var string
+	 */
+	const SIGN_IN_WITH_GOOGLE_USER_ID_OPTION = 'googlesitekitpersistent_sign_in_with_google_user_id';
```

Comment:
@benbowler, actually, I think we can keep it like this since we have the user options instance provided to the module. Let's not introduce a new class for it.